designer/client/src/actions/notificationActions.tsx

```diff
@@ -1,29 +1,48 @@
 import React from "react";
 import Notifications from "react-notification-system-redux";
-import TipsError from "../assets/img/icons/tipsError.svg";
```

Comment:
To be consistent, maybe we can change these icons in other places? If it's too much work, let's do a task to do it, because I feel that we will forget about it :).

Reply:
hm, what do you mean in other places?

Reply:
I meant that we use these icon icons in a few places, not only in the notification component, so to be consistent, maybe it's worth switching them to the material icons everywhere.
![image](https://github.com/TouK/nussknacker/assets/9945753/5d1ca9e7-62e1-40b2-a9c3-5a95c6cdf116)

Reply:
Do we have the option to switch themes?

Reply:
Since we have a stylus, no, but we will want it in the future.